kedro/templates/project/hooks/utils.py

```diff
@@ -105,11 +105,11 @@ def _remove_file(path: Path) -> None:
         path.unlink()
 
 
-def _handle_starter_setup(selected_add_ons_list: str, python_package_name: str) -> None:
+def _remove_PySpark_Viz_starter_files(is_Viz: bool, python_package_name: str) -> None:
```

Comment:
Nit: it looks a bit weird to have the capital letters in the function name. I'd personally just keep it lower case. Same for the `is_Viz` argument